pkgs/top-level/all-packages.nix

```diff
@@ -35630,4 +35630,7 @@ with pkgs;
   honeyvent = callPackage ../servers/tracing/honeycomb/honeyvent { };
 
   mictray = callPackage ../tools/audio/mictray { };
+
+  trino-cli = callPackage ../development/tools/database/trino-cli { };
```

Comment:
Please don't put new entries at the bottom of the file. You can grep for the first couple of letters of trino to see where it should belong. Ideally alphabetically sorted

Reply:
Yeah, it's messy, but we can't fix it without breaking all PRs touching that file.

In the meantime putting them all at the bottom of the file is the worst option. 
There's also another PR that was opened today that did. 
If we merged either, the other one would have a merge conflict. 
Better to disperse it